nginx/metadata.csv

```diff
@@ -3,103 +3,177 @@ nginx.net.writing,gauge,,connection,,The number of connections waiting on upstre
+nginx.cache.miss.bytes_written_count,count,,byte,,The total number of bytes written to the cache (shown as count),0,nginx,cache miss bytes_written
 nginx.cache.miss.responses,gauge,,response,,The total number of responses not taken from the cache,0,nginx,cache miss responses
+nginx.cache.miss.responses_count,count,,response,,The total number of responses not taken from the cache (shown as count),0,nginx,cache miss responses
 nginx.cache.miss.responses_written,gauge,,response,,The total number of responses written to the cache,0,nginx,cache miss responses_written
+nginx.cache.miss.responses_written_count,count,,response,,The total number of responses written to the cache,0,nginx,cache miss responses_written
 nginx.cache.revalidated.bytes,gauge,,byte,,The total number of bytes read from the cache,0,nginx,cache revalidated bytes
+nginx.cache.revalidated.bytes_count,count,,byte,,The total number of bytes read from the cache (shown as count),0,nginx,cache revalidated bytes
 nginx.cache.revalidated.response,gauge,,responses,,The total number of responses read from the cache,0,nginx,cache revalidated responses
+nginx.cache.revalidated.response_count,count,,responses,,The total number of responses read from the cache (shown as count),0,nginx,cache revalidated responses
 nginx.cache.size,gauge,,response,,The current size of the cache,0,nginx,cache size
 nginx.cache.stale.bytes,gauge,,byte,,The total number of bytes read from the cache,0,nginx,cache stale bytes
+nginx.cache.stale.bytes_count,count,,byte,,The total number of bytes read from the cache (shown as count),0,nginx,cache stale bytes
 nginx.cache.stale.responses,gauge,,response,,The total number of responses read from the cache,0,nginx,cache stale responses
+nginx.cache.stale.responses_count,count,,response,,The total number of responses read from the cache (shown as count),0,nginx,cache stale responses
 nginx.cache.updating.bytes,gauge,,byte,,The total number of bytes read from the cache,0,nginx,cache updating bytes
+nginx.cache.updating.bytes_count,count,,byte,,The total number of bytes read from the cache (shown as count),0,nginx,cache updating bytes
 nginx.cache.updating.responses,gauge,,response,,The total number of responses read from the cache,0,nginx,cache updating responses
+nginx.cache.updating.responses_count,count,,response,,The total number of responses read from the cache (shown as count),0,nginx,cache updating responses
 nginx.connections.accepted,gauge,,connection,,The total number of accepted client connections.,0,nginx,conns accepted
+nginx.connections.accepted_count,count,,connection,,The total number of accepted client connections (shown as count).,0,nginx,conns accepted
 nginx.connections.active,gauge,,connection,,The current number of active client connections.,0,nginx,conns active
+nginx.connections.active_count,count,,connection,,The current number of active client connections (shown as count).,0,nginx,conns active
```

Comment:
I don't think this one should be a count: current number of active connections, it can increase or decrease